src/sentry/models/externalactor.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from enum import Enum
+from typing import Optional
+
+from django.db import models
+
+from sentry.db.models import BoundedPositiveIntegerField, DefaultFieldsModel, FlexibleForeignKey
+
+logger = logging.getLogger(__name__)
+
+
+class ExternalProviders(Enum):
+    GITHUB = 0
+    GITLAB = 1
+    EMAIL = 100
+    SLACK = 110
+
+
+EXTERNAL_PROVIDERS = {
+    ExternalProviders.GITHUB: "github",
+    ExternalProviders.GITLAB: "gitlab",
+    ExternalProviders.EMAIL: "email",
+    ExternalProviders.SLACK: "slack",
+}
+
+
+def get_provider_name(value: int) -> Optional[str]:
+    return EXTERNAL_PROVIDERS.get(ExternalProviders(value))
+
+
+class ExternalProviderMixin:
```

Comment:
We had a mixin to duplicate logic for two classes. Now that theses are one model you can remove it.

Reply:
Actually we don't even need these two functions so you should probably just remove this class.